### ISSUE TITLE
Explanation of multi-user GID / UID

### ISSUE THREAD

**Issue by m-graf** [mbentley/docker-timemachine#31]
Please excuse my ignorance. When creating a multi-user config, should the user's created match system (non docker) users? Would I just fetch their UID and place it in the config, and add the user to the `docker` group so that they can have the correct permissions? Or should I use the user's `GID` – and ensure that the user is part of the `docker` group?

For example, if I have my 'timemachine' user who I set to 1000:1000 (which maps to my docker user) should I create additional non-login users on my host, add them to the docker group, and then specify their GID / UID's in the multi-user config? 

Thank you in advance.

**Comment by mbentley** [comment]
@m-graf - so mostly, the UID/GID mapping doesn't really matter a whole lot.  The only thing to really consider would be that if those UID/GID mappings existing on your host, the users and groups that they belong to would be able to access the stored data in the persistent data volume based off the UID/GID mappings.  Typically I would say that there is not really a need from a time machine perspective for users to be able to directly access their data on the time machine server directly but I am sure someone out there will prove me wrong with some use case.

The whole point of the user UID/GID mapping is to provide access control to who can back up data to which share so if you had multiple users backing up their data to the same time machine running on a host, the users would only be able to see their own shares so that one user couldn't back up, delete, or restore to another user's share.

**Comment by m-graf** [comment]
I honestly didn't expect a comment so kind. Thank you. I ended up experimenting with it a little more and digging through the samba logs - it looks like I have to use a custom config. I found a gist of an smb.conf that someone created which modifies some of the default config and corrected my Auth issues. I'll edit this post and post it here when I'm in front of it.

I can create a PR to enable a "best of both worlds" approach; where we can merge a custom smb.conf with the custom user config option- which would allow the users to be properly appended to the smb.conf - is there any interest in that?
